src/frontend/components/InstructorView/index.tsx

```diff
@@ -2,14 +2,13 @@ import { Button } from 'grommet';
 import { normalizeColor } from 'grommet/utils';
 import * as React from 'react';
```

Comment:
This is minor but I think it's better to use `import React from 'react'`. It's more standard and clearer when we also import hooks